src/smartpeak/source/core/Utilities.cpp

```diff
@@ -10,6 +10,7 @@
 #include <unordered_set>
 #include <plog/Log.h>
 #include <boost/filesystem.hpp>
+#include <boost/format.hpp>
```

Comment:
great work, are there any alternatives to `boost/format` ? we're trying to move away from boost except for testing for the time being